src/main/configurations/Translate/CreeerZaak_LK01/xsd/AndereZaakBodyToJson.xsd

```diff
@@ -3,15 +3,15 @@
     <xs:element name="container">
         <xs:complexType>
             <xs:sequence>
-                <xs:element ref="relevanteAndereZaken" maxOccurs="unbounded" />
+                <xs:element ref="relevanteAndereZaken" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
We are trying to step away from having proper XSD's just for XML to JSON conversion, due to issues with it contantly tripping over the order of elements. Instead we want to only put the bare minimum in the XSD to help the XML to JSON with arrays. Only elements that should be handled as arrays should be specified in the XSD, surrounded by xs:any processContents="skip". As example: https://vscode.dev/github/ibissource/zaakbrug/blob/master/src/main/configurations/Translate/Zgw/Zaken/Model/PostZgwZaak.xsd#L10